Cargo.toml

```diff
@@ -11,25 +10,40 @@ documentation = "https://www.wpsoftware.net/rustdoc/secp256k1/"
 description = "Fork of Rust bindings for Pieter Wuille's `libsecp256k1` library. Implements ECDSA for the SECG elliptic curve group secp256k1 and related utilities."
 keywords = [ "crypto", "ECDSA", "secp256k1", "libsecp256k1", "bitcoin" ]
 readme = "README.md"
-
 build = "build.rs"
+
 [build-dependencies]
 cc = "1.0"
-cfg-if = "0.1"
+cfg-if = { version = "0.1" }
 
 [lib]
 name = "secp256k1"
 path = "src/lib.rs"
-
-[features]
-unstable = []
-default = []
-dev = ["clippy"]
+# native platform target
+crate-type = ["lib"]
+# wasm32-unknown-unknown target
+#crate-type = ["cdylib"]
 
 [dependencies]
-arrayvec = "0.4"
-clippy = {version = "0.0", optional = true}
-rand = "0.4"
+sr-std = { git = "https://github.com/chainx-org/sr-std", default-features = false }
```

Comment:
I think most of the occurrences of `rstd` you can just replace by `core::X`. Thus we don't need to pull in an external dependency.